src/quest_manager/models.py

```diff
@@ -975,3 +988,55 @@ def get_minutes_to_complete(self):
 
         minutes = (self.first_time_completed - self.timestamp).total_seconds() / 60
         return minutes
+
+
+class QuestionType(models.Model):
+    id = models.CharField(
+        max_length=100,
+        primary_key=True,
+        help_text="Short, memorable, unique identifier for a question type e.g. 'short_answer', 'file_upload', 'multiple_choice'.",
+        serialize=False
+    )
+    name = models.CharField(
+        max_length=255,
+        help_text="The name of the question type e.g. 'Short Answer', 'File Upload', 'Multiple Choice'."
+    )
+    description = models.TextField(blank=True, null=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Question(models.Model):
+    instructions = models.TextField()
+    solution_text = models.TextField(null=True, blank=True)
```

Comment:
What's the difference between solution text and marker notes?

Reply:
Quite a while ago we had a brief conversation about this: https://github.com/bytedeck/bytedeck/issues/1304#issuecomment-1585349776.

I suppose the idea is that teachers could insert the actual answer to a text-based answer to the question in `solution_text`, or a file answer in `solution_file`. And then the teacher would be able to add additional notes in `marker_notes`. 

For example, the instructions would be "what is the best video game?", the solution_text would be the explicit solution e.g. "Tetris", and the marker_notes might elaborate e.g. "The quest hints towards tetris being the best video game".

If this doesn't make sense, maybe either marker_notes or solution_text could be scrapped?

Reply:
Makes sense!